Lib/test/test_inspect.py

```diff
@@ -233,8 +233,35 @@ class NotFuture: pass
         coro.close(); gen_coro.close() # silence warnings
 
     def test_isroutine(self):
-        self.assertTrue(inspect.isroutine(mod.spam))
+        # to method
```

Comment:
```suggestion
        # methods
```

Also for similar comments below.

Reply:
I've fixed all the typos here.